app/scripts/core/pipeline.py

```
@@ -0,0 +1,871 @@
+
+    vision = None
+    vision_init_error = None
+    if vision_provider != "none":
+        try:
+            vision = VisionDescriber(
+                provider=vision_provider,
+                model=vision_model,
+                max_tokens=int(cfg["vision_max_tokens"]),
+                api_keys=api_keys,
+            )
+        except Exception as exc:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)